File: PatientRecords.BLLayer/Extends/ExtendModelClasses/AuthResponseModel.cs
using PatientRecords.BLLayer.EntityViews;

namespace PatientRecords.BLLayer.Extends.ExtendModelClasses
{
    public class AuthResponseModel
    {
        public bool IsAuthSuccessful { get; set; }
        public string ErrorMessage { get; set; }
        public string Token { get; set; }
        public UserView LoggedUser { get; set; }
    }
}

File: PatientRecords.ReactJS/src/redux/auth/models/authResponseModel.js
import { BaseModel } from 'sjs-base-model';
import userView from '../../users/models/userView';

export default class AuthResponseModel extends BaseModel {
    id = 0;
    isAuthSuccessful =false;
    token = '';
    errorMessage = '';
    loggedUser =  userView;
    constructor(data) {
      super();
      this.update(data);
    }
  }